client/src/app/app.component.html

```diff
@@ -1,2 +1,7 @@
-<router-outlet></router-outlet>
-<app-mentorship-app-root></app-mentorship-app-root>
\ No newline at end of file
+<app-mentorship-app-root></app-mentorship-app-root>
+<nav>
+  <a routerLink="/dashboard"> Dashboard</a>
+  <a routerLink="/login"> Login</a>
+  <a routerLink="/user-profile"> User Profile</a>
+</nav>
```

Comment:
please remove extra pages and add buttons.You can rename one of these to "Advice Wall" or create new.